src/pool/srv_pool_map.c

```diff
@@ -261,30 +267,33 @@ update_one_tgt(struct pool_map *map, struct pool_target *target,
 			break;
 		case PO_COMP_ST_UP:
 			if (target->ta_comp.co_fseq == 1) {
-				D_DEBUG(DB_MD, "change "DF_TARGET" to NEW %p\n",
-					DP_TARGET(target), map);
+				D_DEBUG(DB_MD, DF_MAP ": change " DF_TARGET " to NEW\n",
+					DP_MAP(pool_uuid, map), DP_TARGET(target));
 				target->ta_comp.co_status = PO_COMP_ST_NEW;
 				target->ta_comp.co_in_ver = 0;
 				++(*version);
 			} else {
-				D_DEBUG(DB_MD, "change "DF_TARGET" to DOWNOUT %p fseq %u\n",
-					DP_TARGET(target), map, target->ta_comp.co_fseq);
+				D_DEBUG(DB_MD, DF_MAP ": change " DF_TARGET " to DOWNOUT fseq %u\n",
```

Comment:
```suggestion
				D_DEBUG(DB_MD, DF_MAP ": change " DF_TARGET " to DOWN or DOWNOUT fseq %u\n",
```

Reply:
Changed this to two messages, one for each `if` branch. Please check.